packages/ui/src/hooks/_utils/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export { useForceUpdate } from './useForceUpdate'
```

Comment:
Why separate from `@open-condo/ui/src/components/_utils/hooks` ?

Reply:
Maybe put `_utils` folder on the level above? `src/_utils`?